ocs_ci/utility/performance_dashboard.py

```diff
@@ -59,7 +59,7 @@ def initialize_data():
         ocs_ver = m.group(1)
     log.info(f'ocs_ver is {ocs_ver_full}')
     platform = config.ENV_DATA['platform']
-    if platform.lower() not in ['vsphere', 'baremetal']:
+    if platform.lower() not in ['VSPHERE', 'BAREMETAL']:
```

Comment:
I think the 'if condition' will never be met as `platform.lower()` is always lowercase?

Reply:
Thanks for your comment, I will re-visit the code

Reply:
this has been removed